Tools/LyTestTools/ly_test_tools/o3de/editor_test_utils.py

```diff
@@ -105,7 +106,7 @@ def retrieve_crash_output(run_id: int, workspace: ly_test_tools._internal.manage
         with open(crash_log) as f:
             crash_info = f.read()
     except Exception as ex:
-        crash_info += f"\n{str(ex)}"
+        crash_info += f"\n{str(exceptions.LyTestToolsException(ex))}"
```

Comment:
This should not be a LTT exception because it's the actual crash inside of the test